example/src/screens/CustomSubtitleOnlyUI.tsx

```diff
@@ -0,0 +1,243 @@
+import React, { useCallback, useState } from 'react';
```

Comment:
This example file is effectively what Angel Studios has running in production.  I almost wonder if it would make more sense to put this logic in the PlayerView file itself.  I chose not to do that in order to avoid future painful merge conflicts when I pull in the latest release into our forked repo.  I also chose not to do it because I wasn't sure how to expose the android only subtitle configuration options, without making the API even more weird than it is currently.